src/opm/output/eclipse/AggregateNetworkData.cpp

```diff
@@ -407,8 +409,7 @@ int numberOfBranchesConnToNode(const Opm::Schedule& sched, const std::string& no
         noBranches = (network.uptree_branch(nodeName).has_value()) ? noBranches+1 : noBranches;
         return noBranches;
     } else {
-        auto msg = fmt::format("Actual node: {} has not been defined at report time: {} ", nodeName, lookup_step+1);
-        throw std::logic_error(msg);
+        return 0; // @TODO - Should number of branches in inactive subtree be reported?
```

Comment:
Similar to one of the previous comments, dangling nodes should not matter for the function `network.has_node(nodeName)`, not sure whether this change is necessary.

Reply:
I think you're right, will revert this one.

Reply:
Here, we should keep the throw, I think.

Reply:
Ok!